### PR TITLE
docs/variants/protectli_ptx01: add initial documentation

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: docs/variants/protectli_ptx01/overview.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Overview
+
+The Vault is a small form network appliance built for use as a firewall /
+router, virtualization platform, a daily-driven personal computer, and more.
```
Comment (mkopec) [inline docs/variants/protectli_ptx01/overview.md:4]:
```suggestion
router, virtualization platform, a daily-driver personal computer, and more.
```

File: docs/variants/protectli_ptx01/overview.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+The PTx01 features an Intel jasper Lake CPU, soldered down LPDDR4X memory,
```
Comment (mkopec) [inline docs/variants/protectli_ptx01/overview.md:5]:
```suggestion
The PTx01 features an Intel Jasper Lake CPU, soldered down LPDDR4X memory,
```

File: docs/variants/protectli_ptx01/overview.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Intel i225-V/i226-V 2.5G Ethernet ports, 4x USB-A 3.x ports, eMMC, PCIe NVMe
+storage, M.2 WIFI and WWAN slots, serial console over USB-C, firmware TPM.
```
Comment (mkopec) [inline docs/variants/protectli_ptx01/overview.md:7]:
```suggestion
storage, M.2 WIFI and WWAN slots, serial console over USB-C and a firmware TPM.
```

File: docs/variants/protectli_ptx01/overview.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+* PT201 - Intel® Celeron® N5105, 2x i225/i226, 4GB LPDDR4X
+* PT401 - Intel® Celeron® N5105, 4x i225/i226, 8GB LPDDR4X
+* PT601 - Intel® Pentium® Silver N6005, 6x i225/i226, 16GB LPDDR4X
+
+## Documentation sections
+
+* [Releases](releases.md) - groups information about all releases.
+* [Building manual](building-manual.md) - describes how to build Dasharo for
+  NovaCustom NV4x.
```
Comment (mkopec) [inline docs/variants/protectli_ptx01/overview.md:17]: Should be Protectli

File: docs/variants/protectli_ptx01/overview.md (hunk 1 above)
Comment (miczyg1) [reply docs/variants/protectli_ptx01/overview.md:4]: Daily-driven is correct. It is sourced from Protectli page: https://eu.protectli.com/product/vp2410/

File: docs/variants/protectli_ptx01/overview.md (hunk 3 above)
Comment (miczyg1) [reply docs/variants/protectli_ptx01/overview.md:7]: Fixed: bad49e371bf67722e975b3125d269b57dbe3ce59

File: docs/variants/protectli_ptx01/overview.md (hunk 2 above)
Comment (miczyg1) [reply docs/variants/protectli_ptx01/overview.md:5]: Fixed: bad49e371bf67722e975b3125d269b57dbe3ce59

File: docs/variants/protectli_ptx01/initial-deployment.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+2. Perform Dasharo installation.
+
+This will flash the BIOS region only. After the successful operation, DTS will
+reboot the platform.
+
+This concludes Dasharo deployment process using DTS.
+
+## Build flashrom
+
+Currently, the latest flashrom release lacks support for Comet Lake U internal
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/protectli_ptx01/initial-deployment.md:23]: Fixed: Fixed: bad49e371bf67722e975b3125d269b57dbe3ce59

**Review by macpijan — APPROVED** [review]

**Pull request merged**